appserver/admin/gf_template_web/pom.xml

```diff
@@ -124,12 +124,11 @@
                             <attach>false</attach>
                             <appendAssemblyId>false</appendAssemblyId>
                             <useProjectArtifact>false</useProjectArtifact>
-                        </configuration>                        
+                        </configuration>
                     </execution>
                 </executions>
             </plugin>
 
-            <!-- Validate XML resources -->
```

Comment:
Why are comments being removed?

Reply:
This comment only duplicated information already visible in the id: <id>validate-xml-files</id>